packages/optimizer/spec/transformers/SeparateKeyframes/doesnt_move_invalid_keyframes/input.html

```diff
@@ -0,0 +1,36 @@
+<!doctype html>
+<html ⚡>
+<head>
+  <script async custom-element=amp-experiment src=https://cdn.ampproject.org/v0/amp-experiment-0.1.js></script>
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+  <link rel=stylesheet href=https://cdn.ampproject.org/v0.css>
+  <link href=https://fonts.googleapis.com/css?foobar rel=stylesheet type=text/css>
+  <link href=https://example.com/favicon.ico rel=icon>
+  <style amp-custom>
+    .another{ display:block }
+    @keyframes anim1{
```

Comment:
suggestion: name the keyframe in a way to identify to expected behavior, e.g. `@keyframes invalidHeight`.

Reply:
Good idea, updated